vars/main.yml

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: MIT
+---
+# Put the role's internal variables here that are not distribution specific.
+# You can override these by defining the same variable with a different
+# value in a platform/version specific file in vars/
+
+# Examples of non-distribution specific (generic) internal variables:
+
```

Comment:
even though the role only has 1 main package and 1 selinux package, I would still define them here as variables
* for consistency with other roles - all roles define these as lists of strings
* so that we can refer to them in different contexts rather than having to hardcode the package names everywhere

```yaml
__fapolicyd_packages: [fapolicyd]
__fapolicyd_selinux_packages: [fapolicyd-selinux]
```
I don't think you need the other vars/$platform.yml files - you can remove them